core/src/main/java/com/sequenceiq/cloudbreak/service/existingstackfix/UnboundRestartFixService.java

```diff
@@ -0,0 +1,96 @@
+package com.sequenceiq.cloudbreak.service.existingstackfix;
+
+import static com.sequenceiq.cloudbreak.domain.stack.StackFix.StackFixType.UNBOUND_RESTART;
+
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.dyngr.Polling;
+import com.dyngr.core.AttemptResults;
+import com.sequenceiq.cloudbreak.auth.ThreadBasedUserCrnProvider;
+import com.sequenceiq.cloudbreak.cloud.model.Image;
+import com.sequenceiq.cloudbreak.core.CloudbreakImageNotFoundException;
+import com.sequenceiq.cloudbreak.domain.stack.Stack;
+import com.sequenceiq.cloudbreak.domain.stack.StackFix;
+import com.sequenceiq.cloudbreak.domain.stack.instance.InstanceMetaData;
+import com.sequenceiq.cloudbreak.service.cluster.flow.ClusterOperationService;
+import com.sequenceiq.cloudbreak.service.stack.StackImageService;
+import com.sequenceiq.flow.api.model.FlowCheckResponse;
+import com.sequenceiq.flow.api.model.FlowIdentifier;
+import com.sequenceiq.flow.service.FlowService;
+
+@Service
+public class UnboundRestartFixService extends ExistingStackFixService {
+
+    static final String AFFECTED_STACK_VERSION = "7.2.11";
+
+    static final Set<String> AFFECTED_IMAGE_IDS = Set.of(
+            "26cd5a65-cd5c-457d-8d48-9caf1a486516",
+            "19cf97b8-56d8-4be9-b317-998eea99d884",
+            "c24acec3-9110-4474-9082-3620deac0910"
+    );
```

Comment:
should be private, and if you need the values in the tests please copy them. if I mess them up here the unit test would pass too, and we want to ensure the test would catch that case